etc/spack/defaults/packages.yaml

```diff
@@ -24,12 +24,15 @@ packages:
       elf: [elfutils]
       fftw-api: [fftw, amdfftw]
       flame: [libflame, amdlibflame]
+      fortran-rt: [gcc-runtime, intel-oneapi-runtime]
```

Comment:
Is the consequence of this that every compiler for fortran must provide a separate runtime package that provides it? Are there cases where there's basically nothing to provide at all?

Reply:
> Is the consequence of this that every compiler for fortran must provide a separate runtime package that provides it? 

Currently it's like that, but nothing prevents not to use the runtimes. Ultimately it's up to the compilers to decide which rules apply when they get used. If we have a compiler that doesn't need a separate runtime, we can encode that.

To be clear, right now it's the compilers that say "each package needing fortran and compiling with me needs `fortran-rt`, and I am the provider of it".

Reply:
I'm not seeing why the preference here would mandate a compiler to have a runtime class -- seems up to how we want to model things, so I agree w/Max.